package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.1.0",
   "description": "Blowfish encryption library for browsers and Node.js",
   "main": "dist/blowfish.js",
+  "types": "./egoroof-blowfish.d.ts",
```

Comment:
maybe just `blowfish.d.ts` ?

Reply:
It's the file name of the module's .d.ts, just like `main` for commonjs.

If change this field, we need to change the file name of `egoroof-blowfish.d.ts` to `blowfish.d.ts` also.

And it effects nothing, I don't think it is necessary.

Reply:
yeah, I mean to change filename, as it seems unnecessary long

but it's not blocker, anyway